### ISSUE TITLE
[Accessibility]In case if the Menu item in Commandbar is a drop down (having submenuprops), then it is narrated as "Submenu". It should be narrated as "Menu"

### ISSUE THREAD

**Issue by v-khkuma** [OfficeDev/office-ui-fabric-react#8532]
Bug Report

-Package version(s): https://developer.microsoft.com/en-us/fabric#/components/commandbar

Browser and OS versions: Chrome+NVDA
Actual Behaviour:
In case if the Menu item in Commandbar is a drop down (having submenuprops), then it is narrated as "Submenu"

Expected:
If the Menu item in Commandbar is a drop down (having submenuprops), then it is narrated as "Submenu".
It should be narrated as "Menu"

If applicable, please provide a codepen repro:
`https://codepen.io/micahgodbolt/pen/zaGwee`

**Comment by v-khkuma** [comment]
Any updates on this issue?

**Comment by dzearing** [comment]
@v-khkuma is there an incorrect role? I'm not exactly sure what causes NVDA to distinguish this.

**Comment by v-khkuma** [comment]
No there is no role assigned. Although if we are passing role to the CommandBar, then it should be aasigned to all of them and not only first Menu.
It is behaving same in the official site as well. 
https://developer.microsoft.com/en-us/fabric#/components/commandbar

**Comment by cschlechty** [comment]
The command bar is a type of menu, with menu items, so all subsequent menus will be submenus, no?

**Comment by dzearing** [comment]
@joschect can you help take a look here?

**Comment by joschect** [comment]
@v-khkuma I believe that this is working as intended according to the [aria specs](https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html#)

Just to make sure that we're on the same page though, do you mean that the menu itself is read as a submenu or the actual command bar item/button is read as a submenu? If it's the popup menu itself then it is considered to a be a submenu for the purposes of navigation.
